exporter/coralogixexporter/internal/coralogixpb/coralogixpb.go

```diff
@@ -0,0 +1,847 @@
+// Code generated by protoc-gen-gogo. DO NOT EDIT.
+// source: collector.proto
```

Comment:
iam not able to find `collector.proto` in the entire organisation. https://github.com/search?q=org%3Acoralogix+collector.proto

Reply:
I believe they meant this: https://github.com/jaegertracing/jaeger-idl/blob/main/proto/api_v2/collector.proto

Reply:
yes seems like a modified version of it. https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/7931#issuecomment-1200304790